tutorials/vue/vue-chartjs.md

```diff
@@ -0,0 +1,26 @@
+Chart.js is a great library for visualizing data and displaying it in your projects. To use it with your Vuejs projects,
+there is an excellent wrapper called `vue-chartjs`. You need both `chart.js` and `vue-chartjs` if you wish to use this
+library in your Vuejs projects. If you do not wish to install the libraries, you can instead use it directly in the
+browser via CDN. Because `vue-chartjs` is a wrapper of `chart.js`, every chart type is available for use when working
+with these libraries. When you import one of these chart types from `vue-chartjs`, you need to extend it. You can do so
+with the `extends` property or the `mixins` property. If you choose to use the `mixins` property, you must enclose the import
+with brackets. To create the chart, you must call `this.renderChart()` in the `mounted()` hook where the parameters for `this.renderChart()`
+are the data and any options you provide which you can hardcode or pass in as props. If you hardcode it, the component becomes nonreusable
```

Comment:
I'd get rid of this last sentence about hardcoding it, doesn't seem to provide much value